### PR TITLE
[BACKPORT] Fix dequantizelinear definition

### PR DESCRIPTION
**Pull request by CharlieL7**
Backport of #1567. Trying to get into 6.2.1 release.

### CODE REVIEW

**Review by krzysz00 — APPROVED** [review]
Approved subject to the note that you'll need to bump the commit pointer to the release branch after this lands

**Comment by krzysz00** [comment]
@manupak Can we get a second approval here?

**Review by manupak — APPROVED** [review]

**Pull request merged**
